build/dev-env.sh

```diff
@@ -45,9 +45,11 @@ if ! command -v helm &> /dev/null; then
   exit 1
 fi
 
+function ver { printf "%03d%03d%03d%03d" $(echo "$1" | tr '.' ' '); }
+
 HELM_VERSION=$(helm version 2>&1 | cut -f1 -d"," | grep -oE '[0-9]+\.[0-9]+\.[0-9]+') || true
 echo $HELM_VERSION
-if [[ ${HELM_VERSION} -lt 3.10.0 ]]; then
+if [[ $(ver $HELM_VERSION) -lt $(ver "3.10.0") ]]; then
```

Comment:
Thank you for noticing this issue. However, when I use this PR, I get the following results.
```
➜  ingress-nginx git:(fix-10293) ✗ LANG=EN make dev-env                     
3.8.1                                                                       
build/dev-env.sh: line 52: [[: 003008001000: value too great for base (error token is "003008001000")
```

Reply:
I think, since we are comparing with version 3.10.0, can we do it through the following method?

```
if [[ $(echo -e "$HELM_VERSION\n3.10.0" | sort -V | head -n 1) == "3.10.0" ]]; then

```

Just provide a train of thought, and you can also have different implementation methods.

Reply:
@tao12345666333  Thank for your test. The issue is from leading zero. Can you help me check with this ```function ver { printf "%d%03d%03d" $(echo "$1" | tr '.' ' '); }```
I think use `ver` is better for reading code easily